src/group/GroupView.spec.ts

```diff
@@ -0,0 +1,36 @@
+import * as chai from "chai";
+import { GroupView } from "./GroupView";
+
+describe("GroupView", () => {
+    const view = new GroupView(
+        {
+            1: { id: 1, name: "org", scheme_id: 1},
```

Comment:
probably group as the name rather than org

Reply:
but this is taking a organisation object which is then used to link it in the group